tarofreighter/chain_porter.go

```diff
@@ -705,17 +632,6 @@ func (p *ChainPorter) stateStep(currentPkg sendPackage) (*sendPackage, error) {
 	// send, so we'll make a split with our root change output and the rest
 	// of the created outputs.
 	case SendStatePreparedSplit:
-		preparedSpend, err := taroscript.PrepareAssetSplitSpend(
-			*currentPkg.ReceiverAddr, currentPkg.InputAssetPrevID,
-			*currentPkg.SenderScriptKey.PubKey, *currentPkg.SendDelta,
-		)
-		if err != nil {
-			return nil, fmt.Errorf("unable to create split "+
-				"commit: %w", err)
-		}
-
-		currentPkg.SendDelta = preparedSpend
-
```

Comment:
Looks like we can go ahead and just remove some of these states now as well.

Reply:
Yep, see one of the later commits (in the new PRs).